.gitmodules

```diff
@@ -12,3 +12,6 @@
 [submodule "libdeflate"]
 	path = libdeflate
 	url = ../libdeflate
+[submodule "zstd"]
+	path = zstd
+	url = git@github.com:facebook/zstd.git
```

Comment:
Can't we use the distribution-packaged zstd? I can add it to the frozen toolchain.

Reply:
Ah, you explain why we can't. Please point the submodule at ../zstd, this way we are insulated from changes in facebook's repository. I'll fork it into scylladb/zstd.

Reply:
Forked.

Reply:
Done